gef-ui/server/utils/helpers.js

```diff
@@ -147,6 +147,7 @@ const previewItemConditions = (previewParams) => {
   let summaryItems = [];
   const statusMIA = [
     CONSTANTS.DEAL_STATUS.READY_FOR_APPROVAL,
+    CONSTANTS.DEAL_STATUS.SUBMITTED_TO_UKEF,
```

Comment:
test?

Reply:
Should I add some? None exist for this function I believe